dbt/models/default/schema.yml

```diff
@@ -706,3 +706,130 @@ models:
       # TODO: Data completeness correlates with availability of spatial data
       # by year
   - name: default.vw_pin_exempt
+
+exposures:
```

Comment:
See the docs on [exposures](https://docs.getdbt.com/docs/build/exposures) for more details about what these properties mean.